### PR TITLE
dead_sqrtf, data migration, etc

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀 This PR matches 2 functions (+0.08%) on `us`.

✅ This PR fixes 5 warnings!

File: src/1a1f0_len_5390.c [hunk 1]
```diff
@@ -1,12 +1,174 @@
...
+
+void spawn_drops(Enemy* enemy);
+
+s32 D_80077C40 = 0;
+
+EvtScript D_80077C44 = {
+    EVT_WAIT(10)
+    EVT_CALL(FadeBackgroundToBlack)
+    EVT_WAIT(10)
+    EVT_CALL(CreateNpc, -10, 12255233)
```
Comment (HailSanta) [inline src/1a1f0_len_5390.c:27]: ANIM 0xBB0001 here

File: src/1a1f0_len_5390.c [hunk 2]
```diff
@@ -1,12 +1,174 @@
...
+    EVT_CALL(SetNpcFlagBits, NPC_BTL_MERLEE, NPC_FLAG_100, TRUE)
+    EVT_CALL(SetNpcYaw, NPC_BTL_MERLEE, 0)
+    EVT_CALL(GetCamLookAtObjVector)
+    EVT_CALL(SetNpcPos, NPC_BTL_MERLEE, LocalVar(0), LocalVar(1), LocalVar(2))
+    EVT_THREAD
+        EVT_CALL(MerleeUpdateFX)
+    EVT_END_THREAD
+    EVT_CALL(FadeInMerlee)
+    EVT_WAIT(30)
+    EVT_CALL(SetNpcAnimation, NPC_BTL_MERLEE, 0xBB0000)
```
Comment (HailSanta) [inline src/1a1f0_len_5390.c:37]: ANIM_BattleMerlee_Release

File: src/1a1f0_len_5390.c [hunk 3]
```diff
@@ -1,12 +1,174 @@
...
+    EVT_CALL(MerleeStopFX)
+    EVT_CALL(UnfadeBackgroundFromBlack)
+    EVT_WAIT(20)
+    EVT_THREAD
+        EVT_CALL(FadeOutMerlee)
+        EVT_CALL(DeleteNpc, -10)
```
Comment (HailSanta) [inline src/1a1f0_len_5390.c:43]: NPC_BTL_MERLEE

File: src/1a1f0_len_5390.c [hunk 4]
```diff
@@ -1,12 +1,174 @@
...
+    EVT_END_THREAD
+    EVT_CALL(PlaySound, 0x2075)
```
Comment (HailSanta) [inline src/1a1f0_len_5390.c:45]: SOUND_2075

**Review by HailSanta — APPROVED** [review]

File: src/1a1f0_len_5390.c [hunk 5]
```diff
@@ -1,12 +1,175 @@
...
+            .generalChance = 3,
+            .attempts = 0,
+            .chancePerAttempt = 0,
+        },
+    },
+    .minCoinBonus = 0,
+    .maxCoinBonus = 0,
+};
+
+extern EvtScript D_800936DC;
```
Comment (nanaian) [inline src/1a1f0_len_5390.c:129]: Top of file?

File: tools/build/configure.py [hunk 6]
```diff
@@ -123,6 +123,11 @@ def write_ninja_rules(ninja: ninja_syntax.Writer, cpp: str, cppflags: str, extra
         deps="gcc",
     )
 
+    ninja.rule("dead_cc",
+        description="dead_cc $in",
+        command=f"mips-linux-gnu-objcopy --redefine-sym sqrtf=dead_sqrtf $in $out",
```
Comment (nanaian) [inline tools/build/configure.py:128]: Can we call this rule `dead_objcopy` or something? It isn't cc at least

Comment (nanaian) [reply tools/build/configure.py:128]: Perhaps instead of dead.h we should do `--redefine-sym` for all the dead syms here in the future :thinking:

**Pull request merged**